handbook/sales/sales.rituals.yml

```diff
@@ -1,7 +1,13 @@
 # https://github.com/fleetdm/fleet/pull/13084
 
 
-
+ -
+  task: "Close leads contacted ≥7 days ago"
+  startedOn: "2024-07-05"
+  frequency: "Daily"
+  description: "Close all of your leads in the 'Attempted to contact' stage and which have been there for 7 or more days.  If follow-up is appropriate, and won't be bothersome, it can be done after closing the lead.  (A new lead can always be opened for the contact later.)"
+  moreInfoUrl: ""
+  dri: "Every AE"
```

Comment:
Anthony Snyder :house:  8 minutes ago
I’m not sure I’d want leads to be auto closed, I think I’d always want to manually close out leads when I’m finished reaching out to them. But maybe they just trigger in a dashboard that requires action after 7 days


mikermcneil :mag:  [< 1 minute ago](https://fleetdm.slack.com/archives/C058S8PFSK0/p1720219743273859?thread_ts=1720197343.457819&cid=C058S8PFSK0)
You're right.  That's what this does ^.  Not an automation, but setting the expectation that we're checking every day for 7-day old leads in that stage.
Meanwhile,  there's about to be way fewer leads.  
[@eashaw](https://fleetdm.slack.com/team/U02CUTHQGUC)
 and I just got together to plan out disabling creation of "Signup" leads altogether to stem the bleed.
Instead, Fleet can build the plumbing and programs to nurture them as contacts before automatically sending them to anyone as leads.  (fyi 
[@Drew Baker](https://fleetdm.slack.com/team/U02BD0GCD0T)
 re: what we started last week)
That should take effect in the next few days.  We'll have a dashboard ready that reports on hot "pre-lead" contacts by Monday


@Drew-P-drawers @eashaw @Sampfluger88 @AnthonySnyder8 @alexmitchelliii